src/DirectoryObject.js

```diff
@@ -132,14 +132,18 @@ export class DirectoryObject extends Component {
                         &nbsp;<input id="mountedPath" value={this.state.mountInfo.path} />
                         <Button size="sm" variant="primary" onClick={this.copyPath} ><FontAwesomeIcon icon={faCopy} /></Button>
                     </> : <>
-                        <Button size="sm" variant="primary" onClick={this.mount} >Mount</Button>
+                        <Button size="sm" variant="primary" onClick={this.mount} >Mount Snapshot as Local Drive</Button>
```

Comment:
"Drive" is a Windows Term, Linux/macOS uses mountpoints.

Perhaps "Mount as Local Filesystem" ?

Reply:
Im ok with either "Local Directory" or "Local Filesystem". My intention behind the proposed changes was to better differentiate between mounting / restoring full snapshots vs individual file restores.